### DISCUSSION TITLE
Is it possible to customize `codeMirrorPlugin`?

### DISCUSSION THREAD

**Discussion by melihogul** [mdx-editor/editor#415] [Q&A]
I want to assign "h-20 overflow-y-auto" etc. style classes to the codeMirror plugin. Is there any way to do this?

![image](https://github.com/mdx-editor/editor/assets/87611039/b2e1b0c0-7e6b-4627-97e8-23d0a4f1b04b)

**Comment by iSuslov** [comment]
@petyosi it looks like CodeMirror is one of the most complex elements of this Editor, exposing more customization functionality seems very logical.
